projects/plugins/jetpack/modules/sso/class.jetpack-sso-user-admin.php

```diff
@@ -457,6 +497,10 @@ public function send_wpcom_mail_user_invite( $errors, $update, $user ) {
 			if ( ! $update ) {
 				$valid_nonce = isset( $_POST['_wpnonce_create-user'] ) ? wp_verify_nonce( $_POST['_wpnonce_create-user'], 'create-user' ) : false; // phpcs:ignore WordPress.Security.ValidatedSanitizedInput -- WP core doesn't pre-sanitize nonces either.
 
+				if ( empty( $_POST['invite_user_wpcom'] ) ) {
```

Comment:
Can we use here the `send_wp_mail_new_user` function? (After renaming it) So we have a centralized place for that check